connector/dialect.tdd

```diff
@@ -1033,104 +826,18 @@
     <aggregation value='AGG_MAX'/>
     <aggregation value='AGG_STDEV'/>
     <aggregation value='AGG_STDEVP'/>
-    <aggregation value='AGG_VAR'/>
-    <aggregation value='AGG_VARP'/>
-    <aggregation value='AGG_COVAR'/>
-    <aggregation value='AGG_COVARP'/>
-    <aggregation value='AGG_CORR'/>
-    <aggregation value='AGG_SUM_XSQR'/>
-    <aggregation value='AGG_COLLECT'/>
-
     <aggregation value='AGG_YEAR'/>
-    <aggregation value='AGG_QTR'/>
     <aggregation value='AGG_MONTH'/>
     <aggregation value='AGG_DAY'/>
-    <aggregation value='AGG_WEEK'/>
-    <aggregation value='AGG_WEEKDAY'/>
-    <aggregation value='AGG_MONTHYEAR'/>
-    <aggregation value='AGG_MDY'/>
     <aggregation value='AGG_HOUR'/>
     <aggregation value='AGG_MINUTE'/>
     <aggregation value='AGG_SECOND'/>
```

Comment:
I haven't been able to successfully test aggregating by datetime.  I will open an investigation ticket to look into this further.

Reply:
Opened [SQL-788](https://jira.mongodb.org/browse/SQL-788)